services/reminder.service.js

```diff
@@ -0,0 +1,59 @@
+import {
+  createReminder,
+  fetchReminders,
+  fetchReminder,
+  updateReminder,
+  deleteReminder
+} from '../repository/index.js'
+import responseMessages from '../utils/responseMessages.js'
+
+// Create reminder service logic
+export const createReminderService = async (data) => {
+  try {
+    const { reminderTitle, startDate, endDate, customQuote, challenge, diary } = data
+    await createReminder({ reminderTitle, startDate, endDate, customQuote, challenge, diary })
+    return Promise.resolve(responseMessages.SAVE_SUCCESS)
+  } catch (error) {
```

Comment:
No need try-catch block here in service layer, because you're already handling exceptions in repository layer. @dulshan-perera